### ISSUE TITLE
DNS resolver is not cleared on application shutdown

### ISSUE THREAD

**Issue by jornane** [eduvpn/macos#245]
This issue has occurred for me sporadically earlier, but today I upgraded to MacOS 10.14.6 and the issue seems to be more reproducable.

When I am connected to eduVPN and I quit the application through normal means (tray icon -> Quit eduVPN), the DNS settings on the interface are not cleared.

**Comment by efef** [comment]
we are aware of DNS instabilities in the start/stop scripts. I consider the only stable solution to move to the usage of network-extensions API. Wee're putting effort in porting the iOS app to MacOS. At the moment this works for 10.15. Strangely we've an ipv4 issue on 10.14. I expect to have a MacOS Let's Connect app in the appstore on short notice.

**Comment by bencomp** [comment]
Yesterday I realised that I couldn't use my work MacBook on my home network because the DNS resolver that my network supplies (192.168.100.1, i.e. the router) through DHCP is not in the list of resolvers, and the resolvers that are in the list are not available from my home. Once I manually added the router to the list of DNS resolvers, I was able to use the Internet again.

I'm also on MacOS 10.14.6, but this happens also when I don't quit the application and just disconnect within the app.

**Comment by bencomp** [comment]
My issue appears to be (very similar to) #174. Could it be related?

**Comment by bencomp** [comment]
It looks like #83 is related.

**Comment by jornane** [comment]
Yes, those are related. I just opened a new issue because the problem is still there and the old issues are not being reopened.
